pages/common/rip.md

```diff
@@ -0,0 +1,28 @@
+# rip
+
+> Remove files or directories by sending them to the graveyard, allowing for them to be recovered.
+> More information: <https://github.com/nivekuil/rip>.
+
+- Remove files or directories from specified locations and place them in the graveyard:
+
+`rip {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- Interactively remove files or directories, with a prompt before every removal:
+
+`rip -i {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- List all files and directories in the graveyard that were originally within the current directory:
```

Comment:
```suggestion
- List all files and directories in the graveyard that were originally within the current working directory:
```

Reply:
Personally, I feel like this line is already really long and it might not be worth adding "working" to be more clear. If you disagree (or if you feel that line length doesn't matter at all) I can commit this suggestion.

Reply:
No, it is all fine, I tend to leave further edits which I am not bothered about if they don't get committed after approving!